crates/matrix-sdk-crypto/src/verification/machine.rs

```diff
@@ -651,7 +651,10 @@ mod tests {
         assert!(alice_machine.verifications.outgoing_requests().is_empty());
 
         // This line panics on macOS, so we're disabled for now.
-        alice.set_creation_time(Instant::now() - Duration::from_secs(60 * 15));
+        // Time structs are fully qualified to satisfy "unused imports" lint on macos
+        alice.set_creation_time(
+            matrix_sdk_common::instant::Instant::now() - std::time::Duration::from_secs(60 * 15),
```

Comment:
Instead of using qualified paths, can you just restore the same imports as before, but at the top of this function rather than the `tests` module?